documentation/source/newsfragments/1255.feature.rst

```diff
@@ -0,0 +1,11 @@
+cocotb now supports the following example of forking a non-decorated async coroutine:
+
+.. code-block:: python3
+
+   async def example():
+       for i in range(10):
+           await cocotb.triggers.Timer(10, "ns")
+
+   cocotb.fork(example())
+
+Issue 
```

Comment:
```suggestion
```
CI fails due to the trailing space on line. If this formatting is deliberate, it seems a little weird

Reply:
It's indeed deliberate, because towncrier will append ``(#1255)`` there. Without an extra line with at least a single non-whitespace character, it will just append it to the last code line. :(
towncrier has lots of little warts like that and at the same time is not very actively maintained, so I'm getting doubtful of it in general.
On the other hand, CI should maybe not care about ReST files.

Reply:
I assume that without the trailing space, it also does the wrong thing?

Reply:
Another option would be:
```
Issue (:pr:`1255`)

.. ignore the towncrier issue text
..
```

Reply:
> I assume that without the trailing space, it also does the wrong thing?

It looks like it indeed adds the space itself.